sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/keycloak/KeycloakAdminConsoleSteps.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * SORMAS® - Surveillance Outbreak Response Management & Analysis System
+ * Copyright © 2016-2022 Helmholtz-Zentrum für Infektionsforschung GmbH (HZI)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <https://www.gnu.org/licenses/>.
+ */
+package org.sormas.e2etests.steps.web.application.keycloak;
+
+import static org.sormas.e2etests.pages.application.keycloak.KeycloakAdminConsolePage.NEXT_PAGE_BUTTON;
+import static org.sormas.e2etests.pages.application.keycloak.KeycloakAdminConsolePage.USER_ID;
+import static org.sormas.e2etests.pages.application.keycloak.KeycloakAdminConsolePage.VIEW_ALL_USERS_BUTTON;
+import static org.sormas.e2etests.steps.BaseSteps.locale;
+
+import cucumber.api.java8.En;
+import java.util.concurrent.TimeUnit;
+import javax.inject.Inject;
+import org.sormas.e2etests.envconfig.manager.RunningConfiguration;
+import org.sormas.e2etests.helpers.AssertHelpers;
+import org.sormas.e2etests.helpers.WebDriverHelpers;
+import org.sormas.e2etests.steps.web.application.users.UserManagementSteps;
+import org.testng.Assert;
+
+public class KeycloakAdminConsoleSteps implements En {
+
+  protected WebDriverHelpers webDriverHelpers;
+  int numberOfUsers;
+
+  @Inject
+  public KeycloakAdminConsoleSteps(
+      WebDriverHelpers webDriverHelpers,
+      RunningConfiguration runningConfiguration,
+      AssertHelpers assertHelpers) {
+    this.webDriverHelpers = webDriverHelpers;
+
+    When(
+        "^I navigate to Users tab in Keycloak Administrator Console$",
+        () -> {
+          String KEYCLOAK_USERS_TAB =
+              runningConfiguration.getEnvironmentUrlForMarket(locale)
+                  + "/keycloak/auth/admin/master/console/#/realms/SORMAS/users";
+          webDriverHelpers.accessWebSite(KEYCLOAK_USERS_TAB);
+        });
+    When(
+        "^I click View all users button$",
```

Comment:
mention also Keycloak :D to avoid confusions with actions that we can do in users directory on sormas app